### PR TITLE
Added Query.is_sliced property.

### PR DESCRIPTION
**Pull request by felixxm**
In most of cases we use `Query.can_filter()` to check if a query has limit (is sliced). IMO it's confusing and it will be good to add a `has_limit` property.

### CODE REVIEW

File: django/db/models/sql/query.py [hunk 1]
```diff
@@ -1762,6 +1761,10 @@ def clear_limits(self):
         """Clear any existing limits."""
         self.low_mark, self.high_mark = 0, None
 
+    @property
+    def has_limit(self):
```
Comment (charettes) [inline django/db/models/sql/query.py]: Should we name this `is_sliced` instead? In the case of `[10:]` which translates to `low_mark=10;high_mark=None` there's no `LIMIT` on the query. It also seems more in line with its current usages.

Comment (felixxm) [reply django/db/models/sql/query.py]: Good point, I will amend this later today.

Comment (felixxm) [reply django/db/models/sql/query.py]: Done.

**Comment by felixxm** [comment]
@charettes Thanks!

**Pull request merged**